kuma/static/styles/includes/_vars.scss

```diff
@@ -177,18 +177,18 @@ break points
 - divide pixels by browser default font size (16px)
 - and multiply by 1em to convert unit to em.
 */
-$max-width-default: (1399px / 16px) * 1em;
-$small-desktop-ends: (1199px / 16px) * 1em;
-$tablet-ends: (1023px / 16px) * 1em;
-$mobile-ends: (767px / 16px) * 1em;
-$small-mobile-ends: (479px / 16px) * 1em;
```

Comment:
This was necessary to keep the same general width of our content. I tested the layout using devtools and we seem ok with the minor adjustments to our breakpoints.